### PR TITLE
[tests] Locate all models and data together @open sesame 10/16 18:46

### PR DESCRIPTION
**Pull request by kparichay**
Locate all the models and data together for easier management

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: tests/nnstreamer_filter_python/runTest.sh [hunk 1]
```diff
@@ -47,13 +47,13 @@ popd
 
 # Passthrough test
 export PYTHONPATH=../../build/ext/nnstreamer/tensor_filter/:$PYTHONPATH
-PATH_TO_SCRIPT="python/passthrough.py"
+PATH_TO_SCRIPT="../test_models/models/passthrough.py"
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_python/runTest.sh]: I think it's more like `data` rather than `models`.

Comment (kparichay) [reply tests/nnstreamer_filter_python/runTest.sh]: But this is passed a model file to tensor_filter as it operates on the data passed to tensor_filter.

Comment (dongju-chae) [reply tests/nnstreamer_filter_python/runTest.sh]: Yes... I reused the existing model parameter to support this. Then, it's fine.

Comment (dongju-chae) [reply tests/nnstreamer_filter_python/runTest.sh]: But... it was related to the internal implementation.
I think it's still weird if these example python custom filter scripts are co-located with `*.pb` in `/models/` folder.

Comment (kparichay) [reply tests/nnstreamer_filter_python/runTest.sh]: I think these are models as they provide the specification to take in data, process them and give output, just like .pb file. The only difference is pb file needs a framework like tensorflow to understand them but these .py files are self sufficient.
They much closer to models than data.

Comment (dongju-chae) [reply tests/nnstreamer_filter_python/runTest.sh]: Yes, makes sence

**Pull request merged**